src/postcss/atomic-css-modules/generateScopedName.ts

```diff
@@ -43,10 +43,9 @@ const generateScopedName: generateScopedName = ({
           const key = `${pkg};${name}`;
           trackClasses.set(key, hash);
         }
-
         // short circuit: electron definition for a given name is always
         // highest priority and unique source of truth.
-        return DEVELOPMENT ? `${name.toUpperCase()}_${hash}` : hash;
```

Comment:
this return is in the nested function (wakRules) so it doesn't really need a value, but just returning will not override the key later on in this function